recipe/meta.yaml

```diff
@@ -478,6 +481,43 @@ outputs:
         - test -f $PREFIX/lib/pkgconfig/mkl-sdl.pc  # [linux]
         - if not exist %LIBRARY_LIB%\\pkgconfig\\mkl-sdl.pc exit 1  # [win]
 
+  - name: mkl-static
+    version: {{ mkl_version }}
+    script: repack.sh   # [linux]
+    script: repack.bat  # [win]
+    number: {{ mkl_buildnum|int + dstbuildnum|int }}
+    requirements:
+        {% if win or 'conda-forge' not in my_channel_targets %}
+      run:
+        - {{ pin_subpackage('mkl-include', exact=True) }}
+        - intel-openmp {{ openmp_version.split('.')[0] }}.*
+        - tbb {{ tbb_version.split('.')[0] }}.*
+      {% else %}
+      run:
+        - {{ pin_subpackage('mkl-include', exact=True) }}
+        - llvm-openmp
+        - tbb {{ tbb_version.split('.')[0] }}.*
+      {% endif %}
+        {% if 'conda-forge' not in my_channel_targets %}
+        - blas * mkl
+        {% endif %}
+    about:
+      home: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html
+      summary: Static libraries for Intel® oneMKL libraries
+      description: |
+        Intel® oneAPI Math Kernel Library static libraries for developing software that uses oneMKL
+        This package is a repackaged set of binaries obtained directly from Intel\'s anaconda.org channel.
```

Comment:
I guess it is no longer correct. Same for all other places

Reply:
yes, but woudl like to first deploy and then fix this  - i don't think this is really important